src/super_gradients/training/models/segmentation_models/segformer.py

```diff
@@ -315,6 +315,8 @@ def __init__(
         overlap_patch_stride: List[int],
         overlap_patch_pad: List[int],
         in_channels: int = 3,
+        sliding_window_size: Tuple[int, int] = (1024, 1024),
+        sliding_window_stride: Tuple[int, int] = (768, 768),
```

Comment:
1) If new parameters is added a matching docstring should be also added.
2) (rows, cols) or (width, height) ? That should be reflected in docstring.

Reply:
Fixed on c79d0f2